kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSPropertyAccessorImpl.kt

```diff
@@ -50,23 +51,33 @@ abstract class KSPropertyAccessorImpl(
         get() = TODO("Not yet implemented")
 }
 
-class KSPropertySetterImpl(
+class KSPropertySetterImpl private constructor(
     owner: KSPropertyDeclaration,
     private val setter: KtPropertySetterSymbol
 ) : KSPropertySetter, KSPropertyAccessorImpl(setter, owner) {
+    companion object : KSObjectCache<Pair<KSPropertyDeclaration, KtPropertySetterSymbol>, KSPropertySetterImpl>() {
+        fun getCached(owner: KSPropertyDeclaration, setter: KtPropertySetterSymbol) = cache
```

Comment:
Since repeated, I thought it can be utilized somehow, like:
```kt
<Kt, Ks> KSObjectCache<Kt, Ks> {
  abstract fun Kt.toKs(): Ks
  open fun getCached(kt: Kt): Ks =
    cache.getOrPut(kt) { it.toKs() } // or better to specify ctor?
}
```
but this pair is not trivial perhaps...